src/Api/Memory.Api.ts

```diff
@@ -286,20 +286,20 @@ export default class MemoryApi {
      * @returns Creep[ ]  -- An array of hostile creeps, empty if none
      */
     public static getHostileCreeps(
-        room: Room,
+        roomName: string,
         filterFunction?: (object: Creep) => boolean,
         forceUpdate?: boolean
     ): Creep[] {
         if (
             NO_CACHING_MEMORY ||
             forceUpdate ||
-            !Memory.rooms[room.name].hostiles ||
-            Memory.rooms[room.name].creeps!.cache < Game.time - HCREEP_CACHE_TTL
+            !Memory.rooms[roomName].hostiles ||
```

Comment:
Since we aren't using the room object here, you have to check `!Memory.rooms[roomName] ||` before you can use any properties of the memory object.